app/views/pxe/_template_form.html.haml

```diff
@@ -47,3 +47,5 @@
   %hr
   %h3= _('Script')
   = render :partial => "template_script_data"
+
+= react 'PxeCustomizationTemplateForm', :recordId => @ct.try(:id).try(:to_s)
```

Comment:
Missing newline

Reply:
But ideally you want to replace this at the source, where this gets rendered from, like you did in the other one. Then you can delete this file completely.

Reply:
I kept it for now for reference